extensions/info/deployment/src/main/java/io/quarkus/info/deployment/InfoDevUiProcessor.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.info.deployment;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.LaunchModeBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.info.runtime.InfoRecorder;
+import io.quarkus.vertx.http.deployment.NonApplicationRootPathBuildItem;
+import io.quarkus.vertx.http.runtime.management.ManagementInterfaceBuildTimeConfig;
+
+/**
+ * This processor is responsible for the dev ui widget.
+ */
+public class InfoDevUiProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    @Record(ExecutionTime.STATIC_INIT)
+    CardPageBuildItem create(NonApplicationRootPathBuildItem nonApplicationRootPathBuildItem,
+            InfoBuildTimeConfig config,
+            ManagementInterfaceBuildTimeConfig managementInterfaceBuildTimeConfig,
+            LaunchModeBuildItem launchModeBuildItem,
+            InfoRecorder unused) {
+        CardPageBuildItem pageBuildItem = new CardPageBuildItem();
+
+        var path = nonApplicationRootPathBuildItem.resolveManagementPath(config.path(),
+                managementInterfaceBuildTimeConfig, launchModeBuildItem);
+        pageBuildItem.addPage(Page.externalPageBuilder("App Information")
+                .icon("font-awesome-solid:circle-info")
+                .url(path)
+                .isJsonContent());
```

Comment:
I know it's too late for this, but shouldn't this method be `asJsonContent()`?

Reply:
No, that is giving the client a hint on what the content type is. If it's not set, the ui makes a header call to get the content type. So this is to bypass that header call if we know the type. See https://github.com/quarkusio/quarkus/blob/a4f329be8b23e99c0a9620d3f9d1dcaf6a51eaff/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-external-page.js#L62